src/NuGet.Clients/NuGet.PackageManagement.UI/UserInterfaceService/INuGetUI.cs

```diff
@@ -109,4 +109,4 @@ public interface INuGetUI
 
         Configuration.ISettings Settings { get; }
     }
-}
\ No newline at end of file
+}
```

Comment:
Revert this sicne it's an unnecessary change.

Reply:
VS automatically adds a newline at the end of each file. Should I remove them from other files as well? Or just this one since it's the only change to this file?

Reply:
Just this one because it's the only change.